.github/workflows/prod-test.yml

```diff
@@ -24,6 +24,23 @@ jobs:
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
 
+      - name: Cache Build Results
+        # Prevent error on re-run
+        # https://github.com/actions/cache/issues/107#issuecomment-556037278
+        if: github.event_name == 'push' || github.event_name == 'pull_request'
+        id: build-cache
+        uses: actions/cache@v1
+        with:
+          path: .cache
+          key: v1-build-cache-${{ hashFiles('yarn.lock') }}-${{ github.ref }}-
```

Comment:
The hash not really necessary, but the cache can't be deleted/updated we need a way to update the cache. https://github.com/actions/cache/issues/2